two-factor.php

```diff
@@ -70,7 +72,7 @@ function wpcom_enable_two_factor_plugin() {
 		wpcom_vip_load_two_factor_plugin();
 	}
 }
-add_action( 'setup_theme', 'wpcom_enable_two_factor_plugin' );
+add_action( 'set_current_user', 'wpcom_enable_two_factor_plugin' );
```

Comment:
Should we load the plugin on `plugins_loaded` and then delay adding the notice/filter caps until `set_current_user`? Feels that that would avoid any potential load order issues?